tests/features/solution/solution.notification.feature

```diff
@@ -15,12 +15,21 @@ Feature: Solution notification system
       | Benjamin Stevens | BenjaminStevens@example.com |           |
       | Cecelia Kim      | CeceliaKim@example.com      | moderator |
     And the following solutions:
-      | title                 | description           | logo     | banner     | owner          | contact information | state    |
-      | The Time of the Child | The Time of the Child | logo.png | banner.jpg | The Red Search | Jody Buchanan       | proposed |
+      | title                 | description           | logo     | banner     | owner          | contact information | state            |
+      | The Time of the Child | The Time of the Child | logo.png | banner.jpg | The Red Search | Jody Buchanan       | proposed         |
+      | Some Scent            | Some Scent            | logo.png | banner.jpg | The Red Search | Jody Buchanan       | deletion_request |
     And the following solution user memberships:
       | solution              | user             | roles |
       | The Time of the Child | Benjamin Stevens | owner |
+      | Some Scent            | Benjamin Stevens | owner |
 
     # Test validation email.
     When the user "Cecelia Kim" changes the state of the "The Time of the Child" solution to "Validated"
+    Then an email should be sent to "Benjamin Stevens"
```

Comment:
For now, we only have two distinct cases where emails are sent. One is in a transition change and the other one is in a deletion of a solution. There is no point in handling multiple emails right now as it is not testable.
Still, the way the emails are handled in the tests, is by saving all properties of the email so we can later test the content of the email or the id.